core/src/main/scala/org/bitcoins/core/crypto/HashDigest.scala

```diff
@@ -99,6 +99,17 @@ object Sha256DigestBE extends Factory[Sha256DigestBE] {
   * Represents the result of SHA256(SHA256())
   */
 sealed abstract class DoubleSha256Digest extends HashDigest {
+
+  /**
+    * @note The string representation of this is given in
+    *       big endian style. This is in order to make it
+    *       easier to scan logs for mentions of the hash.
+    *       (Bitcoin Core exposes all human facing hashes
+    *       in big endian style).
+    */
+  override def toString(): String =
```

Comment:
This is something you guys should look at. Is this a good idea? IMO yes (kinda obvious, otherwise I wouldn't have implemented it), but I'm open to other opinions